util/headerreader/header_reader.go

```diff
@@ -260,10 +264,13 @@ func (s *HeaderReader) logIfHeaderIsOld() {
 	if storedHeader == nil {
 		return
 	}
-	headerTime := time.Unix(int64(storedHeader.Time), 0)
-	if time.Since(headerTime) >= 5*time.Minute {
-		s.setError(errors.New("latest header is at least 5 minutes old"))
-		log.Warn("latest L1 block is at least 5 minutes old", "l1Block", storedHeader.Number, "l1Timestamp", headerTime)
+	headerTime := time.Since(time.Unix(int64(storedHeader.Time), 0))
+	if headerTime >= s.config().OldHeaderTimeout {
+		s.setError(errors.Errorf("latest header is at least %v old", headerTime))
+		log.Warn(
+			"latest L1 block is old", "l1Block", storedHeader.Number,
+			"l1Timestamp", headerTime, "age", headerTime,
```

Comment:
I think `l1Timestamp` should be different than `age`, probably the former should be `time.Unix(int64(storedHeader.Time), 0)`

Reply:
Ah true, fixed :)